currency/pairs.go

```diff
@@ -355,3 +410,33 @@ func (p Pairs) GetStablesMatch(code Code) Pairs {
 	}
 	return stablePairs
 }
+
+// ValidateAndConform checks for duplications and empty pairs then conforms the
+// entire pairs list to the supplied formatting (unless bypassed).
+// Map[string]bool type is used to make sure delimiters are not included so
+// different formatting entry duplications can be found e.g. `LINKUSDTM21`,
+// `LIN-KUSDTM21` or `LINK-USDTM21 are all the same instances but with different
+// unintentional processes for formatting.
+func (p Pairs) ValidateAndConform(pFmt PairFormat, bypassFormatting bool) (Pairs, error) {
```

Comment:
When there is duplicate _available_ pairs, the duplicates are removed silently, regardless of whether bypass is enabled or not